src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js

```diff
@@ -188,6 +196,7 @@ export const ScenarioManagerTreeList = (props) => {
               nodesExpandedDetailsRef={nodesExpandedDetails}
               scenarioTree={scenarioTree}
               refreshTick={refreshTick}
+              searchQuery={searchText}
```

Comment:
I can't find the prop `searchQuery` in the `ScenarioSortableTree` component, is it a new prop that was not committed in this PR?

Reply:
:woman_facepalming: it's a remaining of different methods to avoid the webapp crash :slightly_smiling_face: thanks !